platform-core/src/main/kotlin/org/dashevo/Client.kt

```diff
@@ -0,0 +1,19 @@
+/**
+ * Copyright (c) 2020-present, Dash Core Group
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+package org.dashevo
+
+import org.bitcoinj.params.EvoNetParams
+import org.bitcoinj.params.MobileDevNetParams
+import org.dashevo.platform.Platform
+
+class Client(network: String) {
+    val platform = Platform(if (network == "testnet") EvoNetParams.get() else MobileDevNetParams.get())
+
+    fun isReady(): Boolean {
```

Comment:
Can we add a //TODO here and maybe add a comment on how this method will work/

Reply:
Yes, I will do that.  We may need a separate story for this.  Currently, the code assumes that if any node is working, but doesn't have a fall back to use a backup node.  additionally, it would be good for to check for several nodes that they are up and running before making queries to them later.

Reply:
TODO comment added to explain what this isReady method will need to do in the future.

https://github.com/dashevo/android-dashpay/pull/1/commits/91543c5e61ac5b91babbcbfd689dd4e962766bdf